src/simulate/simulateConfig.ts

```diff
@@ -9,12 +19,54 @@ export const simulateConfig: OverwritableConfig = {
   trailingRounds: 7,
 };
 
-export const calculateCombinedScore = (sim: SimulationWithoutCombinedScore) => {
-  const normalizedCapacity = sim.final.capacity * 2;
-  const normalizedUserStoryChance =
-    Math.round((sim.final.userStoryChance + 15) / 3) * 2;
+/* Return false to not completely ignore that game regardless of other scores */
+export function considerGame(
+  appState: AppState<GameActionId, GremlinId>,
+  gameState: GameState<GameActionId, GremlinId>,
+): boolean {
+  /* Ignore games with 0 or 1 gremlin */
+  if (ocurredGremlins.get(appState, gameState) < 2) {
+    return false;
+  }
 
-  return normalizedCapacity + normalizedUserStoryChance;
+  return true;
+}
+
+/**
+ * Tells the simulator which scores to calculate and how
+ * @see scorings.tsx
+ */
+export const scorings = {
+  combinedScore,
+  capacity,
+  gremlinChance,
+  userStoryChance,
+  selectedActions,
+  ocurredGremlins,
+  storiesAttempted,
+  storiesCompleted,
+};
+
+type ScoringKey = keyof typeof scorings;
+type SortDir = 'asc' | 'desc';
+type Sort = Partial<Record<ScoringKey, SortDir>>;
+
+/* only create csv output for these scores */
+export const createCSVFor: ScoringKey[] = ['combinedScore'];
```

Comment:
Will only output/update `combinedScore.csv` feel free to add the others back however you like